request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -209,6 +209,33 @@ def get_time_submitted(self, obj):
     def get_owner(self, obj):
         return obj.application.owner
 
+    def response_change(self, request, obj):
+        if "_approve" in request.POST:
+            if obj.is_approvable():
+                return HttpResponseRedirect(
+                    f"{reverse('application_confirm')}?obj_id={obj.id}&action=approval"
+                )
+            else:
+                self.message_user(
+                    request, "This application can't be approved!", messages.ERROR
```

Comment:
Do you explain why? Or will this be part of the application validation logic to be implemented later?